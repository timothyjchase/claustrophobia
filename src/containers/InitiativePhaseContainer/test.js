import React from 'react'
import { InitiativePhaseContainer, mapStateToProps } from './index'

describe('<InitiativePhaseContainer />', () => {
  describe('with scenarioKey = TEST_SCENARIO', () => {
    const props = {
      currentState: {
        oilForYourLamp: false,
      },
      useAuraOfBlessing: jest.fn(),
      useOilForYourLamp: jest.fn(),
      completeInitiativePhase: jest.fn(),
    }
    const component = shallow(<InitiativePhaseContainer {...props} />)

    it('should match snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })
})

describe('mapStateToProps', () => {
  describe('with basic state', () => {
    const state = { current: {} }
    it('should return currentState', () => {
      expect(mapStateToProps(state)).toEqual({ currentState: {} })
    })
  })
})
