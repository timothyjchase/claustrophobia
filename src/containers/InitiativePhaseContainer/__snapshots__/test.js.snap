// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InitiativePhaseContainer /> with scenarioKey = TEST_SCENARIO should match snapshot 1`] = `
<InitiativePhase
  completeInitiativePhase={[MockFunction]}
  oilForYourLamp={false}
  useAuraOfBlessing={[MockFunction]}
  useOilForYourLamp={[MockFunction]}
/>
`;
